lib/ssl/src/ssl_handshake.erl

```diff
@@ -1621,7 +1622,101 @@ get_cert_params(Cert) ->
             _ ->
                 undefined
         end,
-    {SignAlgo, Param, PublicKeyAlgo, RSAKeySize}.
+    Curve = get_ec_curve(TBSCert#'OTPTBSCertificate'.subjectPublicKeyInfo),
+    {SignAlgo, Param, PublicKeyAlgo, RSAKeySize, Curve}.
+
+get_ec_curve(#'OTPSubjectPublicKeyInfo'{
+                algorithm = #'PublicKeyAlgorithm'{
+                               algorithm = {1,2,840,10045,2,1},
```

Comment:
public_key application provides macros for these OIDs  such as ?'id-ecPublicKey' 
you can find them in OTP-PUB-KEY.hrl in public_key's include directory

Reply:
Done.